global.json

```diff
@@ -0,0 +1,8 @@
+{
+    "sdk": {
+        "version": "2.1.401-preview-009081"
```

Comment:
We should be aware that this is going to affect anyone who uses VS in corefx....

Reply:
yes these changes will definitely affect folks dev flow.